app/Http/Controllers/AuthController.php

```diff
@@ -32,19 +33,24 @@ public function login(Request $request)
             'email' => ['required', 'email'],
             'password' => ['required'],
         ]);
-
-        $user = User::where('email', $request->email)->first();
+    
         if (auth()->attempt($formField)) {
-            $token = $user->createToken('auth_token')->accessToken;
+            $user = auth()->user();
+            $scope = ($user->role==UserRole::Admin->value) ? '*' : 'subscriber';
+    
```

Comment:
can you explain line 39

Reply:
I used it to return scopes for various users. If the role of the user is Admin (1) it should return * i.e permission to all routes and just permission to subscriber route if the user role is not Admin.
(UserRole::Admin->value) already created enums for the subscriber and admin roles.